src/FsAutoComplete/LspServers/AdaptiveFSharpLspServer.fs

```diff
@@ -792,6 +792,11 @@ type AdaptiveFSharpLspServer(workspaceLoader: IWorkspaceLoader, lspClient: FShar
     |> Option.orElseWith (fun () ->
       // TODO: Log how many times this kind area gets hit and possibly if this should be rethought
       try
+        logger.info (
+          Log.setMessage "forceFindOpenFileOrRead else - {file}"
+          >> Log.addContextDestructured "file" file
+        )
```

Comment:
We discussed this the other day. The places where this gets used heavily is on `DidSave` where we want to type check all dependent files. Find All References and Rename also hit this part to properly work. Probably not worth refactoring in this PR tho.